db/migrate/20161203210855_create_venues.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVenues < ActiveRecord::Migration[5.0]
+  def change
+    create_table :venues do |t|
+      t.string :name
+      t.string :address
+      t.string :maps_url
```

Comment:
No, rollback one step, change migration, migrate again. And fix all related code